### PR TITLE
fix: Re-export json in typescript deploy-configs

### PR DESCRIPTION
**Pull request by refcell**
**Description**

Updates the `mainnet.ts` deploy-config to re-export the object defined in `mainnet.json` to de-duplicate values ~~and re-define values inline~~.

**Metadata**

- Fixes CLI-3913

### CODE REVIEW

**Comment by tynes** [comment]
This can still result in a diff between the two

**Review by clabby — APPROVED** [review]

File: packages/contracts-bedrock/deploy-config/mainnet.ts [hunk 1]
```diff
@@ -1,59 +1,27 @@
...
+// Re-export the mainnet json as a DeployConfig object.
+//
+// Notice, the following roles in the system are assigned to the foundationMultisig:
+// - finalSystemOwner
+// - controller
+// - portalGuardian
+// - proxyAdminOwner
+// - l2OutputOracleChallenger
+//
+// The following roles are assigned to the feeRecipient:
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/mainnet.ts:17]:
```suggestion
// The following roles are assigned to the same fee recipient:
```

File: packages/contracts-bedrock/deploy-config/mainnet.ts [hunk 2]
```diff
@@ -1,59 +1,27 @@
...
 // unsafe for a production system.
 
-// The following addresses are assigned to multiples roles in the system, therfore we save them
-// as constants to avoid having to change them in multiple places.
-const foundationMultisig = '0xf39fd6e51aad88f6f4ce6ab8827279cfffb92266' // hh test signer 0
-const feeRecipient = '0x70997970C51812dc3A010C7d01b50e0d17dc79C8' // hh test signer 1
-const mintManager = '0x5C4e7Ba1E219E47948e6e3F55019A647bA501005'
+// Re-export the mainnet json as a DeployConfig object.
+//
+// Notice, the following roles in the system are assigned to the foundationMultisig:
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/mainnet.ts:10]:
```suggestion
// Notice, the following roles in the system are assigned to the 
Optimism Foundation Mulitisig:
```

File: packages/contracts-bedrock/deploy-config/mainnet.ts [hunk 3]
```diff
@@ -1,59 +1,27 @@
...
+// - portalGuardian
+// - proxyAdminOwner
+// - l2OutputOracleChallenger
+//
+// The following roles are assigned to the feeRecipient:
+// - baseFeeVaultRecipient
+// - l1FeeVaultRecipient
+// - sequencerFeeVaultRecipient
+//
+// The following role is assigned to the mintManager:
```
Comment (maurelian) [inline packages/contracts-bedrock/deploy-config/mainnet.ts:22]:
```suggestion
// The following role is assigned to the Mint Manager contract:
```

**Review by maurelian — APPROVED** [review]
Comment nit suggestions

**Pull request merged**